templates/travel/partials/popular/popular.snip.html

```diff
@@ -0,0 +1,72 @@
+{{!
+<!---
+Copyright 2016 The AMP Start Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+<!-- Popular -->
+{{#partials.popular}}
+<section class="travel-popular pb4 pt3 relative">
+  <div class="max-width-3 mx-auto px1 md-px2">
+    <h3 class="h1 bold line-height-2 md-hide lg-hide">Top Adventures<br>Near You</h3>
+    <h3 class="h1 bold line-height-2 xs-hide sm-hide center">Top Adventures Near You</h3>
+  </div>
+  <div class="overflow-scroll">
+    <div class="travel-overflow-container">
+      <div class="flex px1 md-px2 mxn1">
+        {{#adventures}}
+        <div class="m1 mt3 mb2">
+          <div class="{{layout-classes}} mb1">
+            <div class="relative travel-results-result">
+              <a class="travel-results-result-link" href="{{href}}">
+                <amp-img
+                  class="block rounded"
+                  width="{{width}}"
+                  height="{{height}}"
+                  src="{{src}}"
+                ></amp-img>
+              </a>
+              {{#new}}
+              <div class="travel-results-result-flags">
+                <div class="p1"><span class="travel-pill bold">NEW</span></div>
+              </div>
+              {{/new}}
+              <div class="travel-results-result-like">
+                <div class="p1">
+                  {{> ../like/like.snip.html}}
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="h2 line-height-2 mb1">
+            <span class="travel-results-result-text">{{text}}</span>
+            <span class="travel-results-result-subtext h3">&bull;</span>
+            <span class="travel-results-result-subtext h3">$&nbsp;</span><span class="black bold">{{price}}</span>
+          </div>
+
+          <div class="h4 line-height-2">
+            <div class="inline-block relative mr1 h3 line-height-2">
+              <div class="travel-results-result-stars">★★★★★</div>
```

Comment:
why is this being repeated, i think just a color change on the same class should do the trick?

Reply:
Done